src/App.js

```diff
@@ -133,43 +149,39 @@ const App = () => {
           <Search searchQuery={searchQuery} setSearchQuery={setSearchQuery} />
         </header>
       </div>
-      <main>
-        <h1 style={{ textAlign: "center" }}>RoboFriends</h1>
-        <div className="main-center">
-          <select
-            className="main-drop-down-box"
-            onChange={(e) => setSortOptionType(e.target.value)}
-          >
-            <option value="id-up">ID Ascend</option>
-            <option value="id-down">ID Descend</option>
-            <option value="name-up">Name Ascend</option>
-            <option value="name-down">Name Descend</option>
-          </select>
-        </div>
-        {isListLoaded ? (
-          <div className="main-container">
-            {error ? (
-              <div>Error: {error.message}</div>
-            ) : (
-              <>
-                <LoadList
-                  Robots={getPaginatedData()}
-                  onButton={buttonOptions}
-                />
-                <Pagination
-                  maxPage={maxPages}
-                  currentPage={currentPage}
-                  setCurrentPage={setCurrentPage}
-                  pageLimit={pagesToShow}
-                />
-              </>
-            )}
+      <Switch>
+        <main>
+          <h1 style={{ textAlign: "center" }}>RoboFriends</h1>
+          <div className="main-center">
+            <SortByDropDown setSortOptionType={setSortOptionType} />
           </div>
-        ) : (
-          <p>Loading...</p>
-        )}
-      </main>
-    </div>
+          {isListLoaded ? (
+            <div className="main-container">
+              {error ? (
+                <div>Error: {error.message}</div>
+              ) : (
+                <>
+                  <Route exact path="/">
```

Comment:
For the home page, let's make it say "Welcome to Robofriends".
Add a button that brings the user to the first page of the list